src/modules/leader-class.js

```diff
@@ -1,63 +1,69 @@
 export default class LeaderBoard {
   // Initializations
   constructor() {
-    this.leader = [];
+    this.url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/tLLmHQIVXj0SbcfXAHdh/scores';
     this.leaderBoard = document.getElementById('leaderboard');
     this.button = document.getElementById('add-scores');
+    this.refreshButton = document.getElementById('refresh');
   }
 
-  // display book
-  displayBooks(book) {
+  // display scores
+  displayScores(score) {
     let bookContainer = '';
-    book.forEach((arrayItem, index) => {
+    score.forEach((arrayItem, index) => {
       const bookContent = `
-    <div id="${index}" class="score-container">${arrayItem.name}: ${arrayItem.score}</div>       
+    <div id="${index}" class="score-container">${arrayItem.user}: ${arrayItem.score}</div>       
     `;
       bookContainer += bookContent;
     });
     this.leaderBoard.innerHTML = bookContainer;
   }
 
-  // get books in html page if it exist in local storage
-  getLocalStorageData() {
-    const data = JSON.parse(localStorage.getItem('leader'));
-    if (data !== null) {
-      this.displayBooks(data);
+  //  post scores to api
+  addScore = async (data) => {
+    const response = await fetch(this.url, {
+      method: 'POST',
+      headers: {
+        'Content-type': 'application/json',
+      },
+      body: JSON.stringify(data),
+    });
+    const result = await response.json();
+    return result;
+  };
 
-      const scoreContent = document.querySelectorAll('.score-container');
-      scoreContent.forEach((item, index) => {
-        if (index % 2 !== 0) {
-          document.getElementById(item.id).style.backgroundColor = '#D0D0D0';
-        }
-      });
-    } else {
-      this.awesomeBooks.style.display = 'none';
+  // get scores from api
+getScores = async () => {
+  const response = await fetch(this.url);
+  const { result } = await response.json().catch((error) => new Error(error));
+  this.displayScores(result);
+  const scoreContent = document.querySelectorAll('.score-container');
+  scoreContent.forEach((item, index) => {
+    if (index % 2 !== 0) {
+      document.getElementById(item.id).style.backgroundColor = '#D0D0D0';
```

Comment:
- [ ] When I try to enter user and scores it is not displayed on the leaderboard, Kindly make sure to fix it.

https://user-images.githubusercontent.com/70269507/180040077-3caab03c-e3ab-4453-b4d6-01e14dff5e49.mov